packages/scene-composer/src/components/Tree/TreeItem.tsx

```diff
@@ -23,33 +28,45 @@ export interface TreeItemProps extends TreeItemInnerProps, ComponentPropsWithRef
 
 const TreeItemInner: FC<TreeItemInnerProps> = ({
   children,
+  label,
   onSelected = /* istanbul ignore next */ () => {},
   selectable = true,
   selected = false,
   className = '',
   onActivated = () => {},
 }) => {
+  const sceneComposerId = useSceneComposerId();
+  const setSelectedSceneNodeRef = useStore(sceneComposerId).getState().setSelectedSceneNodeRef;
+
+  const selectFromChildren = () => {
+    (children as any)?.filter((child) => {
```

Comment:
should this be `forEach`? doesn't seem to return a filtered result

Reply:
Good call out. Will fix.